lib/models/cac_location.rb

```diff
@@ -67,6 +67,11 @@ def self.to_storepoint(locations)
     locations.map(&:to_storepoint)
   end
 
+  def address
```

Comment:
@ivanoblomov `address` is just street address, not the full address.

You should instantiate a `ComponentizedUSAddress` using the geocoder result for the full address and then call it for address / city / state / postcode. See [Listing](https://github.com/Spark-Bio/covid-storepoint/blob/master/lib/test_sites/listing.rb#L31-L45) for an example.